packages/preview/obsidius/0.1.0/typst.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "obsidius"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["@l0drex"]
```

If you want to have a link to your GitHub profile you should do that instead:

```suggestion
authors = ["l0drex <@l0drex>"]
```

---

Ah, I misread the readme. Would be good to have an example there.